### PR TITLE
docs: release notes to support for gp3 storage in databases

### PR DESCRIPTION
**Pull request by vaibhav1805**

### CODE REVIEW

**Comment by vaibhav1805** [comment]
Marking it as draft, Logmon team needs to add docs under metrics.

File: content/en/docs/releasenotes/deployment/mendix-cloud.md [hunk 1]
```diff
@@ -23,6 +23,20 @@ For information on the current status of deployment to Mendix Cloud and any plan
 
 * We added support for an `Origin-Trials` HTTP header in the Developer Portal. To use this header, the customer must first deploy (or redeploy) their app sometime after 26 October, 12:00 PM. If the customer has deployed since this date, then they can simply set the header and restart their app to use the header. For more information, see [HTTP Headers](/developerportal/deploy/environments-details/#http-headers).
 
+### October 25, 2023
+
+#### Improvements
+
+* We added support for `gp3 storage` instances for databases which provide higher baseline storage performance compared to `gp2`. IOPS and Throughput performance will be upgraded according to the table below. From this date, when a customer creates a new DB instance it will be provisioned with a gp3 storage instance if eligible (storage size above 20 GiB). This also includes the scenario where customer's database is recreated.
```
Comment (feddovanede) [inline content/en/docs/releasenotes/deployment/mendix-cloud.md]: Should we add something like "Existing database will be migrated later. More information will follow."?

File: content/en/docs/developerportal/operate/metrics.md [hunk 2]
```diff
@@ -427,6 +427,19 @@ There are two sets of values:
 
 ### 5.11 Database IOPS Burst Balance {#Trends-dbmxdatabaseburstbalance}
 
+{{% alert color="info" %}}
+We are planning to migrate to `gp3` storage instances soon which will provide higher baseline storage performance compared to `gp2` and does not require any burst balance. This change is already available for new databases starting 25th of October, 2023. When a customer creates a new DB instance it will be provisioned with a gp3 storage instance if it meets the eligibility criteria defined below. This also includes the scenario where customer's database is recreated.
```
Comment (feddovanede) [inline content/en/docs/developerportal/operate/metrics.md]: "We are planning to migrate" -> "We are planning to migrate the databases of apps in the Mendix Cloud"?

Comment (udh-mendix) [reply content/en/docs/developerportal/operate/metrics.md]: fixed

File: content/en/docs/releasenotes/deployment/mendix-cloud.md (hunk 1 above)
Comment (vaibhav1805) [reply content/en/docs/releasenotes/deployment/mendix-cloud.md]: Added

**Comment by vaibhav1805** [comment]
@dbreseman this is done at our end. Can you have a look at it?

**Comment by dbreseman** [comment]
@vaibhav1805 Will do—I'll get this merged this morning.

**Pull request merged**